### PR TITLE
Add a fallback to use IP to get county code

### PR DESCRIPTION
**Pull request by roei-autofleet**
## 📝 Jira tickets:
https://autofleet.atlassian.net/browse/AUT-15397

## ℹ️ Change description:
add a fallback to use IP to get county code

### CODE REVIEW

File: examples/client/Locomotion/src/services/MccMnc.js [hunk 1]
```diff
@@ -12,6 +12,16 @@ const getMobileIsoCode = async () => {
   const isoCode = await CarrierInfo.isoCountryCode();
   return isoCode.toUpperCase();
 };
+export const getCountryCodeByIP = async () => {
+  try {
+    const response = await fetch('https://ipapi.co/json/');
```
Comment (sephina) [inline examples/client/Locomotion/src/services/MccMnc.js]: extract to const (maybe env var?)

File: examples/client/Locomotion/src/services/MccMnc.js [hunk 2]
```diff
@@ -12,6 +12,16 @@ const getMobileIsoCode = async () => {
...
+    const data = await response.json();
+    return data?.country;
+  } catch (error) {
+    console.error('Unable to get country code by IP', error);
```
Comment (sephina) [inline examples/client/Locomotion/src/services/MccMnc.js]: prefer mixpanel event

**Review by sephina — COMMENTED** [review]
what is that api? can we count on its reliability?
also merge to version branch and not directly to master

**Comment by roei-autofleet** [comment]
> what is that api? can we count on its reliability? also merge to version branch and not directly to master

Changed to the version
It's a pretty known free API that provide the location of the IP, this way we don't need to ask for location permissions to set the country code

**Review by sephina — APPROVED** [review]

**Pull request closed without merging**